src/Streamly/Coreutils/Mv.hs

```diff
@@ -0,0 +1,39 @@
+-- |
+-- Module      : Streamly.Coreutils.Mv
+-- Copyright   : (c) 2022 Composewell Technologies
+-- License     : Apache-2.0
+-- Maintainer  : streamly@composewell.com
+-- Stability   : experimental
+-- Portability : GHC
+--
+-- Rename source to dest, or move source(s) to directory.
+
+module Streamly.Coreutils.Mv
+    ( Mv
+    , mv
+    , force
+    , createOnly
+    ) where
+
+import Data.Default.Class (Default(..))
+import System.Directory
+    ( renameFile
+    , renameDirectory
+    )
+
+data Mv = Force | CreateOnly
+
+force :: Mv
+force = Force
```

Comment:
`force` should be a modifier of options i.e. `force :: Switch -> Mv -> Mv`. default should be `Off`.